### ISSUE TITLE
Class `DB` not found

### ISSUE THREAD

**Issue by ddcollins** [fico7489/laravel-eloquent-join#108]
We've upgraded `illuminate/support` to `6.0` and the global `\DB` seems to no longer be available.

```
[2020-05-01 10:40:28] local.ERROR: Class 'DB' not found {"exception":"[object] (Error(code: 0): Class 'DB' not found at /app/vendor/fico7489/laravel-eloquent-join/src/EloquentJoinBuilder.php:134)"} 
```

Is there a new version in the works to support `6.0` by chance?

**Comment by fico7489** [comment]
can you provide more info ? DB is part of the laravel not part of the illuminate/support

**Comment by ddcollins** [comment]
> can you provide more info ? DB is part of the laravel not part of the illuminate/support

Ah, so we aren't using Laravel, but instead using a custom framework built with the illuminate packages. The core issue wasn't us updating to the `6.0` illuminate packages, but the update for this package from `4.0.9` to `4.1.1`. 

`4.0.9` does not have the line
```
$grammar = \DB::query()->getGrammar();
```
 in the `EloquentJoinBuilder.php` file.

I was incorrectly thinking `\DB` was available to us prior to the update to `6.0`, but it has never been. That usage was only just added to this project, which then broke our code.
